interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: when we have two+ vars that happen to have the same name, we use 'this' when we are referring to the global variable or the argument
 
-  Researched answer:
+  Researched answer: In most cases, the value of this is determined by how a function is called (runtime binding).
```

Comment:
"this" is something you might want to look back into. It took me a while to get it but "this" refers to the object that the placeholder variable is a part of.  class Person { this.name, this.age, this. height... etc}